File: Practicas_Univeridad/PRCuadro de mensaje/PRCuadro de mensaje/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace PRCuadro_de_mensaje
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void btningresar_Click(object sender, EventArgs e)
        {
            //Declaracio de variables 
            string nombre;
            //Entrada de datos
            nombre = Microsoft.VisualBasic.Interaction.InputBox("Ingrese Nombre: ","Registro de Datos","Juan Carlos",100,0);
            MessageBox.Show("Bienvenido Usuario: " + nombre, "Sistema Administrativo", MessageBoxButtons.OK, MessageBoxIcon.Information);

        }

        private void btnsalir_Click(object sender, EventArgs e)
        {
            DialogResult opcion;
            opcion = MessageBox.Show("Realmente Desea Salir del Programa?","Salir Del Programa",MessageBoxButtons.OKCancel,MessageBoxIcon.Question);
            if(opcion==DialogResult.OK)
            {
                Close();
            }

        }
    }
}

File: Practicas_Univeridad/prCuadroMensaje/prCuadroMensaje/Form1.vb
Public Class Form1
    Private Sub btningresar_Click(sender As Object, e As EventArgs) Handles btningresar.Click
        'Declaracion de variables
        Dim nombre As String
        'Entrada de datos
        nombre = InputBox("Ingrese Nombre", "Registro de Datos", "Nombre", 100, 0)
        MessageBox.Show("Bienvenido Usuario: " & nombre, "Registro de Datos", MessageBoxButtons.OK, MessageBoxIcon.Information)





    End Sub

    Private Sub btnsalir_Click(sender As Object, e As EventArgs) Handles btnsalir.Click
        Dim opcion As DialogResult
        opcion = MessageBox.Show("Realmente desea Salir de la App?", "Salir de la App", MessageBoxButtons.YesNo, MessageBoxIcon.Question)

        If (opcion = Windows.Forms.DialogResult.Yes) Then
            Me.Close()

        End If
    End Sub
End Class
